collectors/alibaba/collector.js

```diff
@@ -31,6 +31,8 @@ var regionEndpointMap = {
         'ap-northeast-1', 'ap-south-1', 'eu-central-1', 'eu-west-1', 'me-east-1']
 };
 
+var apiVersion = '2015-05-01';
```

Comment:
we're removing this from here and adding it to each call object

Reply:
Sure, will update it accordingly.